database/tables/questions.pg

```diff
@@ -2,6 +2,7 @@ columns
     client_files: text[] default ARRAY[]::text[]
     course_id: bigint not null
     deleted_at: timestamp with time zone
+    dependencies: jsonb default '{}'::jsonb
```

Comment:
I generally hate storing JSON blobs in the DB, rather than making a proper table to store the data, but in this case I think it's ok because the server code is going to want this data in the same format as the dependency data from elements. Is that right?

Reply:
Right